begin
	${3}
rescue ${1:Exception} => ${2:e}
end
